File: Navegador VB/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles bBuscar.Click
        System.Diagnostics.Process.Start("https://www.google.com/search?q=" & tBuscar.Text & "&oq=" & tBuscar.Text & "&aqs=chrome..69i57j69i59l2j69i60l2j69i65j69i61l2.592j0j1&sourceid=chrome&ie=UTF-8")

    End Sub

    Private Sub TextBox1_TextChanged(sender As Object, e As EventArgs) Handles tBuscar.TextChanged

    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub bRojo_Click(sender As Object, e As EventArgs) Handles bRojo.Click
        Me.BackColor = Color.Red
    End Sub

    Private Sub bVerde_Click(sender As Object, e As EventArgs) Handles bVerde.Click
        Me.BackColor = Color.Green
    End Sub

    Private Sub bAzul_Click(sender As Object, e As EventArgs) Handles bAzul.Click
        Me.BackColor = Color.Blue
    End Sub

    Private Sub bVer_Click(sender As Object, e As EventArgs) Handles bVer.Click
        bBuscar.Visible = Not bBuscar.Visible
    End Sub

    Private Sub bIr_Click(sender As Object, e As EventArgs) Handles bIr.Click
        If (String.IsNullOrEmpty(tBuscar.Text)) Then
            Return
        End If
        If (tBuscar.Equals("about:blank")) Then
            Return
        End If
        If Not tBuscar.Text.StartsWith("http://") And Not tBuscar.Text.StartsWith("https://") Then
            tBuscar.Text = "http://" + tBuscar.Text

            Try
                WebBrowser1.Navigate(New Uri(tBuscar.Text))
            Catch
                MessageBox.Show("Se produjo un error.")
            End Try

        End If
    End Sub

    Private Sub WebBrowser1_DocumentCompleted(sender As Object, e As WebBrowserDocumentCompletedEventArgs) Handles WebBrowser1.DocumentCompleted

    End Sub
End Class

File: Navegador C#/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Navegador
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }

        private void button1_Click(object sender, EventArgs e)
        {
            System.Diagnostics.Process.Start("https://www.google.com.ar/search?q=" + tBuscar.Text + "&source=hp&ei=02N0YJOONMqx5OUPgbGGgAs&iflsig=AINFCbYAAAAAYHRx498oX_pJq3Ez2Vd0c4m0u7XT2iRh&oq= " + tBuscar.Text + "perro&gs_lcp=Cgdnd3Mtd2l6EAMyCAguELEDEJMCMgUILhCxAzIFCAAQsQMyBQgAELEDMgUILhCxAzIFCAAQsQMyBQguELEDMgUILhCxAzIFCC4QsQMyBQgAELEDOggIABCxAxCDAToLCAAQsQMQxwEQowI6CAguELEDEIMBOgIIADoLCAAQsQMQxwEQrwE6DggAELEDEIMBEMcBEKMCULsMWI8TYNkUaABwAHgAgAFMiAHQApIBATWYAQCgAQGqAQdnd3Mtd2l6&sclient=gws-wiz&ved=0ahUKEwjTw_3B__jvAhXKGLkGHYGYAbAQ4dUDCAc&uact=5");
        }

        private void textBox1_TextChanged(object sender, EventArgs e)
        {

        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void bAzul_Click(object sender, EventArgs e)
        {
            this.BackColor = Color.Blue;
        }

        private void bRojo_Click(object sender, EventArgs e)
        {
            this.BackColor = Color.Red;
        }

        private void bVerde_Click(object sender, EventArgs e)
        {
            this.BackColor = Color.Green;
        }

        private void bVer_Click(object sender, EventArgs e)
        {
            bBuscar.Visible = !bBuscar.Visible;
        }

        private void bIr_Click(object sender, EventArgs e)
        {
            if (String.IsNullOrEmpty(tBuscar.Text))
                return;
            if (tBuscar.Equals("about:blank"))
                return;
            if (!tBuscar.Text.StartsWith("http://") && !tBuscar.Text.StartsWith("https://"))
            {
                tBuscar.Text = "http://" + tBuscar.Text;
            }
            try
            {
                webBrowser1.Navigate(new Uri(tBuscar.Text));
            }
            catch
            {
                MessageBox.Show("Se produjo un error.");
            }
        }

        private void webBrowser1_DocumentCompleted(object sender, WebBrowserDocumentCompletedEventArgs e)
        {

        }
    }
}
